### DISCUSSION TITLE
Is there no X86ToggleTouchpad key?

### DISCUSSION THREAD

**Discussion by txtyash** [kmonad/kmonad#389] [Q&A]
I want to remap a key to the toggle touchpad key. I was able to remap brightness keys, volume keys but not the keyboard backlight key. Also I'm unable to find a toggle touchpad key.

**Comment by jokesper** [comment]
This is probably due to the brightness keys residing on a different device. I don't quite know the reason but some of the keys toggled by fn are part of a different keyboard device and therefore cannot be remapped (trivialy)
